fetch-vault-secrets.js

```diff
@@ -0,0 +1,26 @@
+require('dotenv').config()
+
+const options = {
+    apiVersion: 'v1',
+    endpoint: 'https://vault.stanford.edu',
+  };
+  const vault = require('node-vault')(options);
+  const fs = require('fs');
+
+  const getSecrets = async() => {
+    try {
+      await vault.approleLogin({
+        role_id: process.env.NEXT_PUBLIC_VAULT_ROLE_ID,
+        secret_id: process.env.NEXT_PUBLIC_VAULT_SECRET_ID,
```

Comment:
```suggestion
        role_id: process.env.VAULT_ROLE_ID,
        secret_id: process.env.VAULT_SECRET_ID,
```

We absolutely DO NOT want this secret and key built publicly anywhere. These values must be kept from the public at all times and never built into the outputted code.